### PR TITLE
New tabs in VirtualServer, Cluster, CloudHost

### PR DESCRIPTION
**Pull request by ghost**
`DNS` tab in:
- `VirtualServer`
- `Cluster`

`SecurityInfo` in:
- `VirtualServer`
- `CloudHost`

### CODE REVIEW

File: src/ralph/data_center/views/ui.py [hunk 1]
```diff
@@ -1,21 +1,6 @@
...
 
 
-class DataCenterAssetSecurityInfo(
-    RalphDetailView
-):
-
-    icon = 'lock'
-    label = 'Security Info'
-    name = 'security_info'
+class DataCenterAssetSecurityInfo(SecurityInfo):
```
Comment (mkurek) [inline src/ralph/data_center/views/ui.py]: move it to `data_center.admin`

**Comment by ar4s** [comment]
:ok:

File: src/ralph/security/templates/security/securityinfo/security_info.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{% extends BASE_TEMPLATE %}
```
Comment (mkurek) [inline src/ralph/security/templates/security/securityinfo/security_info.html:1]: delete `/src/ralph/data_center/templates/data_center/datacenterasset/security_info.html`

File: src/ralph/virtual/admin.py [hunk 3]
```diff
@@ -25,6 +28,22 @@
...
+if settings.ENABLE_DNSAAS_INTEGRATION:
+    from ralph.dns.views import DNSView
+
+    class VirtualServerDNSView(DNSView):
+        namespace = None
+
+
+class VirtaulServerSecurityInfoView(SecurityInfo):
+    url_name = 'security_virtualserver_security_info'
+    template_name = 'security/securityinfo/security_info.html'
```
Comment (mkurek) [inline src/ralph/virtual/admin.py]: couldn't it be moved to `SecurityInfo`?

**Comment by mkurek** [comment]
:ok:

**Pull request merged**